docs/docs/search.md

```diff
@@ -0,0 +1,83 @@
+---
+id: search
+title: Search
+---
+
+All resources have search enabled by default. It uses the [pg_search](https://github.com/Casecommons/pg_search) gem under the hood and applies `pg_search_scope` to them.
+Use the param `query` to search all fields that are either `string` or `text`.
+
+```ruby
+module Projects
+  class Resource < ApplicationResource
+  end
+end
+```
+
+```ruby
+  get '/projects?query=baz'
+```
+
+### Customization
+
+Use search_by to configure [pg_search_scope](https://github.com/Casecommons/pg_search#pg_search_scope) manually.
+
+```ruby
+module Lists
+  class Resource < ApplicationResource
+    search_by :search,
```

Comment:
It is not clear to me if the search_by first parameter must be `:search` or if it can be whatever name I like.
If it must be always `:search`, maybe the first parameter may be removed at all.